atman_violet.mk

```diff
@@ -8,17 +8,18 @@ $(call inherit-product, device/xiaomi/violet/device.mk)
 
 # Inherit some common Lineage stuff.
 $(call inherit-product, vendor/aosp/config/common.mk)
+$(call inherit-product, vendor/pubg/config/cheats.mk)
 
 CUSTOM_BUILD_TYPE := FENIXED
 FENIX_OFFICIAL := true
 
-TARGET_BOOT_ANIMATION_RES := 1080
+TARGET_BOOT_ANIMATION_RES := 9999
 
 # Device identifier. This must come after all inclusions.
 PRODUCT_NAME := atman_violet
 PRODUCT_DEVICE := violet5G
-PRODUCT_BRAND := AtmanOS-5G
-PRODUCT_MODEL := ATMANOS NOTE 7 MEME 5G edition
+PRODUCT_BRAND := Xiaomeme
+PRODUCT_MODEL := PUBG MOBILE 7 Pro 5G Edition
```

Comment:
how dare u touch the model meme

Reply:
this is important sar 
without that , we cannot hack **PABG**